src/index.ts

```diff
@@ -111,7 +111,10 @@ function isV2Event(event: GatewayEvent): event is APIGatewayProxyEventV2 {
 
 function normalizeV1Event(event: APIGatewayProxyEvent): HTTPGraphQLRequest {
   const headers = normalizeHeaders(event.headers);
```

Comment:
I think I'd be more interested in seeing this change happen within `normalizeHeaders`. Header keys are case-insensitive, so I think we should just downcase all the incoming header keys using `.toLocaleLowerCase()` as a part of the normalization process. Does that sound reasonable to you?

Reply:
That makes total sense to me and is a much cleaner solution. Thank you for pointing me in the right direction! I just pushed up that change along with a changeset.